tony-history-server/test/controllers/BrowserTest.java

```diff
@@ -14,7 +13,7 @@
 public class BrowserTest extends WithBrowser {
 
   protected Application provideApplication() {
-    return fakeApplication(ImmutableMap.of("tony.configPath", "/dummy"));
+    return fakeApplication(inMemoryDatabase());
```

Comment:
I had to set `tony.configPath` for the test to get the tests to pass. Currently, they seem to be failing: https://travis-ci.org/linkedin/TonY/builds/449003339?utm_source=github_status&utm_medium=notification

Reply:
I can remove the tests for now if that travis build is your concern. Let me know. @erwa

Reply:
I think you changed the value to `tony.historyFolder`. We should keep and fix the tests. I think you need to set a value for the `tony.historyFolder` config or else the test will fail with `com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'tony'` as seen in the Travis CI build.

Reply:
Ok, I'm going to keep the tests, and I should be able to fix the Travis build by today.